net/libndp/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libndp
+PKG_VERSION:=1.6
+PKG_RELEASE:=1
+PKG_LICENSE:=LGPL-2.1
+PKG_MAINTAINER:=Thomas Guyot-Sionnest <tguyot@gmail.com>
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://github.com/jpirko/libndp/archive/
+PKG_HASH:=565d6c4167f83ec697c762ea002f23e8f0b00828d0749b1ce928f068543e5aad
+
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+PKG_FIXUP:=autoreconf
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/libndp
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=NDP Library/Tool
+  URL:=https://github.com/jpirko/libndp/libndp-1.6/
+endef
+
+define Package/libndp/description
+    Library for Neighbor Discovery Protocol
+endef
+
+define Build/Configure
```

Comment:
So you really need this?

Reply:
Probably not with the right tarball... :)